admin/docker-entrypoint.sh

```diff
@@ -8,13 +8,16 @@ http:
     key: ${ONLINECONF_TLS_KEY}
     redirect_from_http: }':80'
   static_root: /var/www/onlineconf-admin
+  ${ONLINECONF_AUTH_HEADER:+behind_reverse_proxy: true}
 database:
   host: ${ONLINECONF_DATABASE_HOST}
   base: ${ONLINECONF_DATABASE_BASE:-onlineconf}
   user: ${ONLINECONF_DATABASE_USER:-onlineconf}
   password: ${ONLINECONF_DATABASE_PASSWORD}
   timeout: 2
 auth:
+  method: ${ONLINECONF_AUTH_HEADER:+'header'
+  header: ${ONLINECONF_AUTH_HEADER}}
   host: ${ONLINECONF_AUTH_HOST:-${ONLINECONF_DATABASE_HOST}}
```

Comment:
Following parameters (`host`-`timeout`) are specific for `method=mysql`.

Reply:
But this work too  ) these lines ignored while method set as header 
I'm fix